src/utils/crt_launch/SConscript

```diff
@@ -1,23 +1,18 @@
-#!python
-# (C) Copyright 2016-2021 Intel Corporation.
-#
-# SPDX-License-Identifier: BSD-2-Clause-Patent
-#
```

Comment:
There were left over in SConscript files copied from cart, most of our files do not have this header and it was out of date anyway.